Dockerfile.template.erb

```diff
@@ -198,6 +198,9 @@ ENV LD_PRELOAD=""
 <% else %>
 ENV LD_PRELOAD="/usr/lib/libjemalloc.so.2"
 <% end %>
+<% if is_alpine || is_debian %>
```

Comment:
Could you add a note for this fix?
Because this hack is needed for Ruby 2.6.7, 2.7.3 and 3.0.1.

Reply:
I've added.